tests/insights-api-tests/advancedAnalytics.spec.ts

```
@@ -11,324 +11,332 @@ type Analytics = {
+        ];
+
+        const responseObj = await sendGraphqlQuery({
+          request,
+          url: process.env.GRAPHQL_ENDPOINT as string,
+          timeout: queryDeadline,
+          query: graphqlQueriesWithGeojson[0],
+          polygon: JSON.stringify(polygon),
+        });
+
+        for (const layer of responseObj?.data?.polygonStatistic?.analytics
+          ?.advancedAnalytics) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)